### ISSUE TITLE
[QUESTION] .DRMX files and DeDRM_tools

### ISSUE THREAD

**Issue by rdlf4** [apprenticeharper/DeDRM_tools#2175]
Hello,
I've purchased some ebooks from a website, which sent me an email message with some .drmx files (I was also sent a license code to open each .drmx file with --- nothing special, just a nine-digit-long word) and the link to download and install Javelin, which has no Linux version. So I went with Calibre. Then, after googling around, I found DeDRM_tools, which I tried to use in order to get rid of the drm and finally access the ebook files.
I'm on Linux Mint and I just want to read my files in peace, with no intrusive windows updates that will take forever --and then some to complete. After following this guide https://www.cloudwards.net/remove-drm-from-kindle-books/ I ended up nowhere. I know this tool/addon is geared more towards Kindle devices, but I wished the 'eReader books' option from DeDRM_tools 1.7.2 would help me out. It didn't. Calibre version 6.9.
Because I own no kindle device and I'm trying to open a .drmx file, the 'eReader books' option is the only one suitable for me.
Regardless, it seems my problem is opening the file if the screenshot below is anything to go by.

![calibreerror](https://user-images.githubusercontent.com/17544190/204962171-6c6606ab-2328-45f9-9d38-c34d33691800.png)

Where to go from here? Would you happen to know of a Calibre add-on that can handle & decrypt .drmx? Any help is appreciated.

**Comment by ElleKayEm** [comment]
I don't believe there's anything here that can help.  This repo isn't maintained anymore.  Current development is here: https://github.com/noDRM/DeDRM_tools

You need 10.0.3 for calibre 6.x.  But these plugins are for Kindle, Adobe, Kobo, and B&N.  There are a few other things for some older formats that aren't really in use anymore.  There are some other odds and ends in Other_Tools: https://github.com/noDRM/DeDRM_tools/tree/master/Other_Tools

But nothing that sounds like this.  I'm not aware of anything else either.

**Comment by rdlf4** [comment]
OK, so Javelin does allow the protected files to be printed, but not to a virtual printer. I've tried three different virtual printer options, none work. Is there anything else that does a decent job at emulating a printer so that the files I paid for gets saved as pdf, without DRM?

**Comment by peter20051** [comment]
> OK, so Javelin does allow the protected files to be printed, but not to a virtual printer. I've tried three different virtual printer options, none work. Is there anything else that does a decent job at emulating a printer so that the files I paid for gets saved as pdf, without DRM?

try searching for help on reddit
